src/content/docs/installation/browser.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Using the nemo playground
+---
+To get started with Nemo without any installation process check out our [interactive browser tool](https://tools.iccl.inf.tu-dresden.de/nemo). It is easy to use and supports code editing: 
+![Nemo web screenshot](../../../assets/nemo-web-screenshot.png)
+
+---
+
+If you want like to reproduce how a certain result is computed, click the grey button to see its trace.
+For example, the image below shows how the fact `commonDescendantsOfIsabelleAndHeinrich(edward)` is computed:
+![Nemo tracing screenshot](../../../assets/tracing-screenshot.png)
```

Comment:
Again, needs better alt text describing what is actually shown there (to make this make sense in context, the alt text needs to actually describe how that fact is derived).